spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -46,24 +46,13 @@ utilization.
 ==== Publishing to a topic
 
 `PubSubTemplate` provides asynchronous methods to publish messages to a Google Cloud Pub/Sub topic.
-It supports different types of payloads, including `Strings` with different encodings, `byte[]`,
-`ByteString`, POJOs, and `PubsubMessage`.
+The `publish()` method takes in a topic name to post the message to, a payload of a generic type and, optionally, a map with the message headers.
 
-[source,java]
-----
-public ListenableFuture<String> publish(String topic, String payload, Map<String, String> headers)
-
-public ListenableFuture<String> publish(String topic, String payload, Map<String, String> headers,
-Charset charset)
-
-public ListenableFuture<String> publish(String topic, byte[] payload, Map<String, String> headers)
-
-public ListenableFuture<String> publish(String topic, ByteString payload, Map<String, String> headers)
-
-public ListenableFuture<String> publish(String topic, PubsubMessage pubsubMessage)
-
-public <T> ListenableFuture<String> publish(String topic, T payload, Map<String, String> headers)
-----
+* If the payload is a string, it is decoded using the active character set.
+** A character set can be set using the `setCharset()` method in `PubSubTemplate`.
```

Comment:
Why double asterisk ?

Reply:
To make it a sub-item of the first one.
It will be a little bit indented and the bullet also looks different.